modules/services/localization/src/main/res/values/strings.xml

```diff
@@ -1592,7 +1600,7 @@
     <!-- Text that appear when someone share the listened numbers story to Twitter. %1$d is a placeholder for the number of podcasts listened and %2$d for the number of episodes. -->
     <string name="end_of_year_story_listened_to_numbers_share_text">I listened to %1$d different podcasts and %2$d episodes in 2022</string>
     <!-- Title for the story that display the most listened podcast by the user this year. %1$s is a placeholder for the podcast title and %2$s is a placeholder for the author. -->
-    <string name="end_of_year_story_top_podcast">%1$s was your most listened show in 2023</string>
+    <string name="end_of_year_story_top_podcast_title">%1$s was your most listened show in 2023</string>
```

Comment:
Be careful in reviewing string updates as the older strings may have different placeholder counts in translations that could cause a crash.

Reply:
Thanks for mentioning this. I wouldn't have thought about it, and now I'll be more careful.